io.edgehog.devicemanager.Battery.json

```diff
@@ -0,0 +1,34 @@
+{
+  "interface_name": "io.edgehog.devicemanager.Battery",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "datastream",
+  "ownership": "device",
+  "aggregation": "object",
+  "mappings": [
+    {
+      "endpoint": "/battery/%{batterySlot}/levelPercentage",
```

Comment:
@rbino should we use `/battery` or `/batteries` ?

Reply:
@bettio third option: what about dropping it? For example we don't have `/sensors/%{sensor_id}` [here](https://github.com/astarte-platform/astarte/blob/master/standard-interfaces/org.astarte-platform.genericsensors.Values.json#L11) or [here](https://github.com/astarte-platform/astarte/blob/master/standard-interfaces/org.astarte-platform.genericsensors.Geolocation.json#L12).

Also, note that we usually use `snake_case` for placeholders (but `camelCase` is accepted too, I checked)